_posts/2020-08-05-Map-Submissions.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Map Submissions
+author: Crazy_
+---

```

Comment:
This blog post is an announcement, which means you should add `categories: announcement` below the `author` tag.